Seeing Travis failures and all this lint failure:
`npm run lint

> cspace-ui-plugin-profile-bampfa@1.0.4 lint /Users/remillet/dev/src/cspace/cspace-ui-plugin-profile-bampfa.js
> eslint --ext .jsx,.js *.js src test examples


/Users/remillet/dev/src/cspace/cspace-ui-plugin-profile-bampfa.js/src/plugins/recordTypes/person/utils.js
  64:54  error  Expected '!==' and instead saw '!='  eqeqeq
  68:68  error  Missing semicolon                    semi

✖ 2 problems (2 errors, 0 warnings)`

--- a/src/plugins/recordTypes/person/utils.js
+++ b/src/plugins/recordTypes/person/utils.js
@@ -61,9 +61,9 @@ export const computePersonNames = ({ data, recordData }) => {
   // replacing if there is a display name already there.
   let finalNameLFM = namePartsLFM.filter(part => !!part).join(' ');
 
-  finalNameLFM = (finalNameLFM === '' && displayName != '') ? displayName : finalNameLFM;
+  finalNameLFM = (finalNameLFM === '' && displayName !== '') ? displayName : finalNameLFM;
 
   return data
     .set('termDisplayName', finalNameLFM)
-    .set('termName', namePartsFML.filter(part => !!part).join(' '))
+    .set('termName', namePartsFML.filter(part => !!part).join(' '));
 };

